packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -160,9 +189,55 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
       address(0)
     );
     emit OracleReported(token, msg.sender, now, value, DENOMINATOR);
+    recomputeRate(token);
+  }
+
+  function fracMulExp(FixidityLib.Fraction memory a, FixidityLib.Fraction memory b, uint256 exp)
```

Comment:
Perhaps include in the fixidity lib?

Reply:
fixidity lib cannot inherit UsingPrecompiles...